testing/content/files/en-us/web/fixable_flaws/wikihistory.json

```diff
@@ -0,0 +1,4 @@
+{
+  "modified": "2019-01-01T01:01:01.550Z",
+  "contributors": ["cmills", "wbamberg"]
+}
```

Comment:
Just a reminder that we should re-base this PR on the `master` branch, and use your new centralized `_wikihistory.json` instead.